snap/hooks/install

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -eux
```

Comment:
Do you need trace (`x`) by default?

Reply:
It's not required, but it is helpful if there was an error.  If the install hook fails, the install fails, otherwise, the output is discarded.